pagopa/pn-frontend
# Test/pn 3865 Cypress Tests for "Contacts" Section

## Short description
add cypress tests for cypress contacts section

## List of changes proposed in this pull request
- add cypress test for legal contact creation
- add cypress test for enable/disable io contact
- add cypress test for courtesy email creation
- add cypress test for courtesy phone number creation
- add cypress test for special contact creation
- add fixtures and test-data mocks for contact section

## How to test
While pf web application running in local development mode launch cypress for pf ('yarn cy-open:pf' or 'yarn cy-run:pf') and verify all tests pass.
